scripts/release/promote/wheel_to_pypi.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "${DIR}/common_utils.sh"
+
+# Allow for users to pass PACKAGE_NAME
+# For use with other packages, i.e. torchvision, etc.
+PACKAGE_NAME=${PACKAGE_NAME:-torch}
+
+pytorch_version="$(get_pytorch_version)"
+
+# This assumes you have already promoted the wheels to stable S3
+pkgs_to_promote=$(\
+    curl -fsSL https://download.pytorch.org/whl/torch_stable.html \
+        | grep "${PACKAGE_NAME}-${pytorch_version}" \
+        | grep -v "%2B" \
```

Comment:
I'm not entirely sure where this `+` get's added in our build process. @soumith would you possibly know?

Reply:
probably here?
https://github.com/pytorch/builder/blob/81306b42b5aa9d1ac73c3948cd96dff43776cb2d/packaging/wheel/linux_manywheel.sh#L11-L15